News_App/src/app/authentication/authentication.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="auth auth__wrapper">
+  <div class="auth" *ngIf="auth.user$ | async as user">
+    <span class="auth__status"> Вы авторизованы как {{ profile.name }}</span>
+    <div>
+      <img
+        class="auth__img"
+        [src]="profile.picture"
+        [alt]="valueAttributeImgAlt"
+      />
+    </div>
+  </div>
+  <div *ngIf="auth.isAuthenticated$ | async">
```

Comment:
You can't use auth service in template/ You should use for it your authetication component

Reply:
corrected